src/osbuild-manifests/coreos.osbuild.aarch64.mpp.yaml

```diff
@@ -32,7 +32,7 @@ mpp-vars:
   # the host (COSA) gets used as the buildroot there.
   buildroot:
     mpp-if: osname in ['rhcos', 'scos']
-    then: "name:build"
+    then: "name:plain-files-tree"
```

Comment:
Hmm, I don't quite understand the `plain-files-tree` naming TBH. :)
I think the thing to emphasize is that the tree represents a "deployed" OSTree? How about `deployment-root`? Or e.g. `deployed-tree`? Or `deployed-ostree`?